pages/osx/zipinfo.md

```diff
@@ -0,0 +1,11 @@
+# zipinfo
+
+> View the contents of a zip file.
+
+- List files contained in a zip file:
+
+`zipinfo {{compressed.zip}}`
+
+- List only file names of zip file contents:
```

Comment:
The first example also mentions "List files". So there is no difference in the descriptions. Perhaps you should say "List file entries and their metadata" in the first example.